### PR TITLE
TACO-205 - Adjusted Tracking Opt In to TCF 2.2

### PR DESCRIPTION
**Pull request by domjanik**
In this PR I've added necessary changes, that should be made in order to stay compliant with new TCF 2.2 regulations. The most important changes are: 
- Added easily visible amount of Vendors on the 1st screen
- Added easily visible amount of Vendors that uses certain purpose
- Added 11th purpose
- Added Examples of each purpose
- Added links to privacy policy and legitimate interests in proper language for each vendor
- Added required Reject All button in every country
- Added information which Data Category and how long each Vendor is using
I've also bumped version of TCF library, and added a handling new version of vendor-list that is required to work with TCF2.2

### CODE REVIEW

File: src/gdpr/ConsentManagementProvider.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { default as installCMPStub } from '@iabtcf/stub';
 
 import { debug, getCookieDomain, getJSON } from '../shared/utils';
 
-export const CMP_VERSION = 5; // Increment to force modal again
+export const CMP_VERSION = 6; // Increment to force modal again
```
Comment (jbienkowski-fandom) [inline src/gdpr/ConsentManagementProvider.js:9]:
```suggestion
export const CMP_VERSION = 7; // Increment to force modal again
```
`6` has already been used by experiment released in PL for Experian (see #306)

File: src/modal/ScreenOne.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ import globalStyles from './styles.scss';
 import styles from './ScreenOne.scss';
 
 import getParagraphs from './utils/getParagraphs';
+import {IAB_VENDORS} from "../shared/consts";
```
Comment (jbienkowski-fandom) [inline src/modal/ScreenOne.js:7]: unused import?

File: src/i18n/pl/tracking-opt-in.json [hunk 3]
```diff
@@ -53,5 +53,7 @@
     "storageDetailsCookieMaxAgeSeconds": "sekund",
     "storageDetailsCookieMaxAgeMinutes": "minut",
     "storageDetailsCookieMaxAgeHours": "godz.",
-    "storageDetailsCookieMaxAgeDays": "dni"
+    "storageDetailsCookieMaxAgeDays": "dni",
+    "ourPartnersCount": ["Poprzez zaakceptowanie naszej %privacyPolicy% zgadzasz się także na wykorzystywanie Twoich danych przez %partnerCount% naszych partnerów."],
```
[the remark this answers is not in the document]
Comment (domjanik) [reply src/i18n/pl/tracking-opt-in.json:57]: These translations will be added in the separate task, as we need to sent them to the translation process ;)

File: src/shared/consts.js [hunk 4]
```diff
@@ -247,6 +247,7 @@ export const IAB_VENDORS = [
     173, // YieldMo
     726, // Yougov
     301, // Zeotap GmbH
+    1237, // Aryel s.r.l
```
[the remark this answers is not in the document]
Comment (domjanik) [reply src/shared/consts.js]: Good catch - it was added only for testing and I forgot to delete it after tests ;)

**Review by jbienkowski-fandom — APPROVED** [review]

**Pull request merged**